less/hotgraphic.less

```diff
@@ -143,6 +143,27 @@
         font-size:0.75em;
         line-height: 1em;
     }
+
+		.hotgraphic-widget {
```

Comment:
There's a lot of unnecessary nesting here. Suggest the selector becomes just:
```less
.numbers .hotgraphic-graphic-pin {
    // ...
}
```
Please also sort indentation.